src/stash/tests/stash.rs

```diff
@@ -313,6 +314,27 @@ async fn test_stash_postgres() {
         assert_eq!(raw_rows.len(), 1);
         assert_eq!(rows, BTreeMap::from([(S2 { a: 1, b: Some(3) }, 2)]));
     }
+
+    // Test large update batches.
```

Comment:
Nice! Is there anyway we can add a test that calls `update(...)` so we can exercise the new code that splits the batches?

Reply:
The test below already does that. It constructs an update with two values of size INSERT_BATCH_SPLIT_SIZE. When their keys are included it exceeds the batch size and gets split (I verified this with some dbg) into two separate queries.

Reply:
Great!